homeassistant/components/zwave_js/services.py

```diff
@@ -428,11 +466,11 @@ async def async_set_value(self, service: ServiceCall) -> None:
             )
 
             if success is False:
-                raise SetValueFailed(
+                raise HomeAssistantError(
```

Comment:
Why do we change this?

Reply:
In testing I realized that the UI will only show the error if it is a HomeAssistantError. I can submit this change as a separate PR if that makes more sense, I just made the change because I noticed it while testing

Reply:
It's a small change. Let's keep it.